src/lib/middleware/errorHandler.tsx

```diff
@@ -69,6 +70,12 @@ export const errorHandlerMiddleware = async (
     console.log(detail)
   }
 
+  // Log server errors (code, stack trace) when in production mode
+  if (NODE_ENV === "production" && code >= 500) {
+    const logger = createLogger("lib/middleware/errorHandlerMiddleware")
```

Comment:
might as well move this line up after line 41 in case we want to use it elsewhere later